Automattic/sensei
# Merge `feature/student-management` branch to `trunk`

### Changes proposed in this Pull Request

* Update Student Management

### Testing instructions

* Go to `Sensei LMS -> Students`
* Make sure everything works as expected :)
